chapters/statements.tex

```diff
@@ -33,10 +33,10 @@ \subsection{Execution of an algorithm in a model}\label{execution-of-an-algorith
 \lstinline!:=! are initialized (at least conceptually):
 \begin{itemize}
 \item
-  A non-discrete variable is initialized with its start value (i.e.\ the
+  A continuous-time variable is initialized with its start value (i.e.\ the
```

Comment:
Aren't we missing parameters in initial algorithms now?

Reply:
I agree that it remains an issue, but I wouldn't put it at that line.

Reply:
So maybe add another item then?

Reply:
Done. But it required a bit more text.